### ISSUE TITLE
[compose] Play once then loop from frame to frame based on boolean

### ISSUE THREAD

**Issue by Iulian-Liviu** [airbnb/lottie-android#2318]
I would love to see a feature that allows playing an animation once and then looping from frame 20 to frame 50, based on a Boolean value or some other method. It would be similar to the looping functionality in the Java version of this library. It would be awesome if this feature could also be implemented in Jetpack Compose.

Another suggestion is to make it easier to integrate Value Animator in Lottie Animation or provide direct arguments to control the behavior of the Lottie animation. Currently, I have considered using AndroidView, but I'm having trouble figuring out how to make it work.

**Comment by gpeal** [comment]
I think you should be able to achieve this with LottieAnimatable:
```
    @Composable
    fun Content() {
        val composition by rememberLottieComposition(LottieCompositionSpec.RawRes(R.raw.heart))
        val lottieAnimatable = remember { LottieAnimatable() }

        LaunchedEffect(Unit) {
            lottieAnimatable.animate(composition)
            while (true) {
                lottieAnimatable.animate(composition, clipSpec = LottieClipSpec.Frame(min = 20, max = 50))
            }
        }
        LottieAnimation(composition, { lottieAnimatable.progress })
    }
```

Would this work?

**Comment by Iulian-Liviu** [comment]
It dosen't work I am getting a warning : Lottie doesn't support layer effects. If you are using them for  fills, strokes, trim paths etc. then try adding them directly as contents  in your shape. Found: []

And also crashes the app.

I am using this [animation](https://lottiefiles.com/132287-gradient-music-mic) but I don't there would be a problem if I manipulating frames.

I have attached a log : [Log.txt](https://github.com/airbnb/lottie-android/files/11780880/Log.txt)
